Mention mirror_percent field in mirroring task (#4975)
istio/istio.io: content/en/docs/tasks/traffic-management/mirroring/index.md

    Also, it is important to note that these requests are mirrored as "fire and
    forget", which means that the responses are discarded.

>     You can make use of the `mirror_percent` field to mirror only a fraction of the traffic,

Reviewer: ```suggestion
    You can use the `mirror_percent` field to mirror a fraction of the traffic,
```

>     instead of mirroring 100% of the requests. If this field is absent - for compatibility with

Reviewer: ```suggestion
    instead of mirroring all requests. If this field is absent, for compatibility with
```

>     older versions - all the traffic will be mirrored.

Reviewer: ```suggestion
    older versions, all traffic will be mirrored.
```
